pkg/api/vlabs/const.go

```diff
@@ -67,3 +71,8 @@ const (
 	// ManagedDisks means that the nodes use managed disks for their os and attached volumes
 	ManagedDisks = "ManagedDisks"
 )
+
+// Network policy
+var (
+	NetworkPolicyValues = [...]string{"", "none", "azure", "calico"}
```

Comment:
Interesting. Never seen it like that anywhere before. I'll defer to you, seems like what you did was appropriate.